user/languages/r.md

```diff
@@ -206,6 +206,10 @@ repos:
   on this one. This can be quite expensive, so it's not recommended to leave
   this set to `true`.
 
+* `disable_homebrew`: if `true` this removes the preinstalled homebrew 
+  installation on OS X. Useful to test if the package builds on a vanilla OS X
+  machine, such as the CRAN mac builder.
```

Comment:
This is a reference to https://cran.r-project.org/ ? Would that be useful to link? I'm leaning more towards removing " such as the CRAN mac builder" entirely unless it adds value. Thoughts?